### ISSUE TITLE
[Bug] DBT Project Evaluator fails to compile with DBT 1.5-rc1

### ISSUE THREAD

**Issue by ttusing** [dbt-labs/dbt-project-evaluator#327]
### Describe the bug
DBT project evaluator 0.4.1 fails to compile with DBT 1.5-rc1.

```
18:17:26  Running with dbt=1.5.0-rc1
18:17:27  Unable to do partial parsing because saved manifest not found. Starting full parse.
18:17:38  Encountered an error:
Parsing Error
  Invalid models config given in models/marts/tests/testing.yml @ models: {'name': 'fct_test_coverage', 'description': 'this model shows a few key metrics related to how well tested your dbt project is.', 'columns': [{'name': 'test_to_model_ratio', 'description': 'the total number of tests in the project divided by the total number of models in the project', 'tests': None}, {'name': 'test_coverage_pct', 'description': 'the number of models in the project with at least one test configured divided by the total number of models in the project', 'tests': [{'dbt_utils.accepted_range': {'name': 'valid_test_coverage', 'min_value': "{{ var('test_coverage_target') }}", 'severity': 'warn'}}]}], 'original_file_path': 'models/marts/tests/testing.yml', 'yaml_key': 'models', 'package_name': 'dbt_project_evaluator'} - at path ['columns'][0]['tests']: None is not of type 'array'
```

### Steps to reproduce
Attempt to build DBT 1.5-rc1.

### Expected results
DBT compile runs without error.

### Actual results
Above error is produced

### Screenshots and log output
See above

### System information
**The contents of your `packages.yml` file:**

**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [x] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
(route66-gu7ql3jY-py3.9) ttusing@KM1-TTUS164 route66-data-platform % dbt --version
Core:
  - installed: 1.5.0-rc1
  - latest:    1.4.5     - Ahead of latest version!

Plugins:
  - snowflake: 1.5.0b4 - Ahead of latest version!
 ```

### Are you interested in contributing the fix?
Yes, I believe removing or expanding the commented out code in the YML will fix this.

**Comment by b-per** [comment]
Hi @ttusing . Thanks for raising it. The fix is already in `main` (with [this commit](https://github.com/dbt-labs/dbt-project-evaluator/commit/9f03a02ed5f0d09d7d5bad3cf64f84d624468cbb)) and we are going to do a new release of the package today or tomorrow.

I'll check with the team to see if we also fix the current minor version

**Comment by ttusing** [comment]
> Hi @ttusing . Thanks for raising it. The fix is already in `main` (with [this commit](https://github.com/dbt-labs/dbt-project-evaluator/commit/9f03a02ed5f0d09d7d5bad3cf64f84d624468cbb)) and we are going to do a new release of the package today or tomorrow.
> 
> I'll check with the team to see if we also fix the current minor version

Thank you! A minor version release would be helpful and making upgrading to DBT 1.5/testing rc1 easier 🙏

**Comment by ttusing** [comment]
This is fixed in release `0.5.0`! 🎊
